packages/web-pkg/src/composables/actions/files/useFileActionsCreateLink.ts

```diff
@@ -30,19 +31,33 @@ export const useFileActionsCreateLink = ({
   const passwordEnforcedCapabilities = useCapabilityFilesSharingPublicPasswordEnforcedFor()
   const { defaultLinkPermissions } = useDefaultLinkPermissions()
   const { createLink } = useCreateLink()
+  const { copyToClipboard } = useClipboard()
 
-  const proceedResult = (result: PromiseSettledResult<Share>[]) => {
+  const proceedResult = async (result: PromiseSettledResult<Share>[]) => {
     const succeeded = result.filter(
       (val): val is PromiseFulfilledResult<Share> => val.status === 'fulfilled'
     )
-    if (succeeded.length && showMessages) {
-      store.dispatch('showMessage', {
-        title: $ngettext(
-          'Link has been created successfully',
-          'Links have been created successfully',
-          succeeded.length
-        )
-      })
+
+    if (succeeded.length) {
+      let successMessage = $gettext('Link has been created successfully')
+
+      if (result.length === 1) {
+        // Only copy to clipboard if the user tries to create one single link
+        try {
+          await copyToClipboard(succeeded[0].value.url)
+          successMessage = $gettext('The link has been copied to your clipboard.')
+        } catch (_) {}
```

Comment:
should we maybe log a warning at least? I feel empty catches are often really hard to debug